extensions/rv32im/circuit/src/adapters/jalr.rs

```diff
@@ -139,6 +144,19 @@ impl<AB: InteractionBuilder> VmAdapterAir<AB> for Rv32JalrAdapterAir {
     }
 }
 
+#[repr(C)]
+#[derive(AlignedBytesBorrow, Debug)]
+pub struct Rv32JalrAdapterRecord {
+    pub from_pc: u32,
+    pub from_timestamp: u32,
+
+    pub rs1_ptr: u32,
+    pub rd_ptr: Option<u32>,
```

Comment:
I think we should avoid `Option` if possible? It has bad alignment. I think we can use `u32::MAX` as an indicator